cmd/cli/main.go

```diff
@@ -153,6 +193,14 @@ func runRollback(cmd *cobra.Command, args []string) {
 }
 
 func init() {
+	getCmd.Flags().StringVarP(&getAuthor, "author", "a", "", "author of changes")
+	getCmd.Flags().StringVarP(&since, "since", "s", "", "start of time range")
+	getCmd.Flags().StringVarP(&until, "until", "u", "", "end of time range")
+	getCmd.Flags().StringVarP(&resource, "resource", "r", "", "resource name to filter by")
+	getCmd.Flags().StringVarP(&namespace, "namespace", "p", "", "namespace to filter by")
+	getCmd.Flags().StringVarP(&name, "name", "n", "", "name to filter by")
```

Comment:
minor thing but maybe we should make namespace the -n flag to keep consistency with kubectl